docs/cookbook/prompt_engineering/chaining_based/least_to_most.md

```diff
@@ -94,8 +92,8 @@ def least_to_most(query_context: str, query_question: str):
         return response
 
 
-query_context = """The Census Bureaus 2006-2010 American Community Survey showed that
-(in 2010 inflation adjustment dollars) median household income was $52,056 and the
+query_context = """The Census Bureaus 2006-2010 American Community Survey showed that\
```

Comment:
this now has no space between "that" and "(in ...)" and same below? make sure to fix this everywhere please